docs/user/library/jdbc/db2.rst

```diff
@@ -69,6 +69,25 @@ For more information check the java docs for:
 * DB2NGJNDIDataStoreFactory
 * DB2NGDataStoreFactory
 
+Advanced Geotools Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++-------------------+------------------------------------------+
+| Parameter         | Description                              |
++===================+==========================================+
+| "loose bbox"      | Flag controlling loose bbox comparisons, |
+|                   | default is true                          |
++-------------------+------------------------------------------+
+| "use selectivity" | Flag to raise the priority of spatial    |
+|                   | indices for the db2 optimizer,           |
+|                   | default is false                         |
++-------------------+------------------------------------------+
```

Comment:
It does help. If SELECTIVITY means something to DB2 users (which it looks like it does since you have it in an SQL example) then I am happy to keep the word.

The description though is about an effect, SELECTIVITY seems to be a detail on how it is implemented?

How about: SELECTIVITY: Value of true will inject ``SELECTIVITY 0.0001`` hint to the optimizer to raise the priority of the spatial index. Default is false.